app/lib/parseOpeningTimes.js

```diff
@@ -50,7 +50,21 @@ function addPadding(parsedTimes) {
   return parsedTimes;
 }
 
+function isOpen(times) {
+  let openDays = 0;
+  daysOfWeek.forEach((day) => {
```

Comment:
includes won't work as that matches an exact value, rather than the output of a function. Have used 'some' instead.